### PR TITLE
[New] jquery.hotkeys

### PR DESCRIPTION
**Pull request by RandomlyKnighted**
This adds the jQuery.Hotkeys library. Files are taken from the [jQuery.Hotkeys releases](https://github.com/jeresig/jquery.hotkeys/releases).

My old PR was done completely with the GitHub website and on the master branch, which caused problems when trying to do everything in A commit. This PR fixes both that and keeps everything in A commit.

I'd also like to note that I cannot enable the git auto-update feature because the repo's author does not include minified files in their releases. I had to manually minify each js file before submitting.

 - [X] Not found on cdnjs repo
 - [X] No already existing issue and PR
 - [X] Notable popularity **Watchers:** 77, **Star:** 2.076, **Fork:** 546
 - [X] Project has public repository on famous online hosting platform (or has been hosted on npm)

### CODE REVIEW

**Comment by Amomo** [comment]
Hi @RandomlyKnighted ,
we use `filename` field in package.json to point to the mainfile of a lib, this field will be required.
And, I didn't find "jquery.hotkeys" on npm, maybe you could use `git` auto-update.
It would be great if you could also add `keywords` in package.json.
Thank you! :-)

**Comment by RandomlyKnighted** [comment]
@Amomo Sorry, I was already in the process of pushing my project.json changes before I saw your message. It may take some time for me to add the keywords to the project.json. My computer is really slow when I'm working in this project. I guess it's due to it's large size. 

The only issue I see with using git auto-update is that as I mentioned the author of this library does not release a minified JS file. I had to manually minify before I submitted my PR. I can submit an issue to the libraries author about adding minified files. Should I wait about the auto-update until after they start adding that?

**Comment by Amomo** [comment]
You can point `filename` to your manually minified file. :-)
Because the library isn't exist on npm, you should not use npm auto-update.

**Comment by RandomlyKnighted** [comment]
Right, I was actually referring to the git auto-update not the npm auto-update.

**Comment by Amomo** [comment]
No, You are using a npm auto-update config.
A git auto-update config is look like this (example from our [README.md](https://github.com/cdnjs/cdnjs/blob/master/README.md)),
```
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/jashkenas/underscore.git",
    "basePath": "",
    "files": [
      "underscore-min.js",
      "underscore-min.map",
      "underscore.js"
    ]
  }
```

**Comment by RandomlyKnighted** [comment]
Ah, I see now. I was just following another project's project.json as an example. I'll go ahead and remove that since I'll need to do a manual update until the project's author starts uploading minified files.

Besides removing the npm auto-update is there anything else I'm doing wrong in my project.json? I have hard time reading the error in the circlci check.

**Comment by maruilian11** [comment]
@RandomlyKnighted you can try [sparseCheckout.md](https://github.com/cdnjs/cdnjs/blob/master/sparseCheckout.md) to help you get fast with our project, thank you for your contribution!

**Comment by RandomlyKnighted** [comment]
@maruilian11 can you tell me what I'm doing wrong in my project.json? I have a PR created for another project and it's waiting on me to add the CDN for this library before it can be merged so I'd like to get this fixed as soon as I can.

**Comment by maruilian11** [comment]
@RandomlyKnighted sure, i'll help you as possible as i can.
i think the problem is the **"jquery.hotkeys"** in `keywords` .

File: ajax/libs/jquery.hotkeys/package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "jquery.hotkeys",
+  "filename": "jquery.hotkeys.min.js",
+  "version": "0.2.0",
+  "description": "jQuery Hotkeys lets you watch for keyboard events anywhere in your code supporting almost any key combination.",
+  "keywords": [
+    "keyboard",
+    "events",
+	"jquery.hotkeys"
```
Comment (maruilian11) [inline ajax/libs/jquery.hotkeys/package.json]: here is the error!

**Comment by RandomlyKnighted** [comment]
@maruilian11 I removed ```jquery.hotkeys``` from the ```keywords``` but that didn't fix it. 

When I run ```npm test``` do I need to add anything else? The reason I ask is because when I run that command I get this message:

```
> jquery.hotkeys@0.2.0 test c:\projects\cdnjs\ajax\libs\jquery.hotkeys
> echo "Error: no test specified"

"Error: no test specified"
```

**Comment by maruilian11** [comment]
@RandomlyKnighted not under `cdnjs\ajax\libs\jquery.hotkeys`, you should run `npm test` just under `cdnjs/`.
https://circleci.com/gh/cdnjs/cdnjs/3936
seems that you have wrong package format, maybe because of some redundant space, try to copy the right format from the result of npm test

hope this can help you :)

**Pull request closed without merging**